dsp/modules/databricks.py

```diff
@@ -52,23 +54,26 @@ def __init__(
             **kwargs,
         )
 
-        self.kwargs.pop('frequency_penalty', None)
-        self.kwargs.pop('presence_penalty', None)    
+        self.kwargs.pop("frequency_penalty", None)
+        self.kwargs.pop("presence_penalty", None)
```

Comment:
is it worth having a seperate "run ruff on everything" PR? The pre-commit hooks only look at changed files so source which predates ruff is still unformatted.